.yarn/patches/@metamask-network-controller-npm-21.0.0-559aa8e395.patch

```diff
@@ -7,26 +7,26 @@ index 0000000000000000000000000000000000000000..78b9156dc2b0bf7c33dadf325cb3ec0b
 +We remove `lookupNetwork` from `initializeProvider` in the network controller to prevent network requests before user onboarding is completed.
```

Comment:
I'm not sure why this is still a patch, but it was here before so I maintained it.  I verified it is still required to avoid requests during onboarding

Reply:
I can confirm that we still need this patch